gamestonk_terminal/cryptocurrency/due_diligence/dd_controller.py

```diff
@@ -167,16 +174,14 @@ def print_help(self):
    change          30d change of supply held on exchange wallets
```

Comment:
I don't think we need to have `chart` again on this menu since it is right above!